templates/ra_dualaz_rdgw_elb.compound

```diff
@@ -137,9 +143,10 @@
                     "Parameters" : 
                     [ 
                         "RdgwAmiNameSearchString",
+                        "AmiId",						
```

Comment:
I didn't notice this before, but there are a handful of lines with trailing tabs/spaces. Like this one.